app/views/organizations/edit.html.erb

```diff
@@ -33,23 +31,37 @@
         <%= f.telephone_field :phone_number, class: "form-control", data: { guard: "required phoneUS" } %>
       </div>
 
-      <div class="row">
-        <div class="col-xs-12 col-sm-9">
-          <div class="form-group">
-            <%= f.label :external_type %>
-            <%= f.select :external_type, options_for_select(external_types_for_select, @organization.external_type), { include_blank: true }, { id: "external-type", class: "required", data: { guard_required_target: "#external-type-error-target" } } %>
+      <% if current_user.can_update_organization_external_and_admin_details? %>
+        <div class="row">
+          <div class="col-xs-12 col-sm-9">
+            <div class="form-group">
+              <%= f.label :external_type %>
+              <%= f.select :external_type, options_for_select(external_types_for_select, @organization.external_type), { include_blank: true }, { id: "external-type", class: "required", data: { guard_required_target: "#external-type-error-target" } } %>
 
-            <div id="external-type-error-target"></div>
+              <div id="external-type-error-target"></div>
+            </div>
           </div>
-        </div>
 
-        <div class="col-xs-12 col-sm-3">
-          <div class="form-group">
-            <%= f.label "External Id" %>
-            <%= f.number_field :external_id, class: "form-control" %>
+          <div class="col-xs-12 col-sm-3">
+            <div class="form-group">
+              <%= f.label "External Id" %>
+              <%= f.number_field :external_id, class: "form-control" %>
+            </div>
+          </div>
+
+          <div class="col-xs-12 col-sm-6">
+            <div class="form-group">
+              <%= f.label "Programs" %>
+              <%= f.select :program_ids,
+                           options_for_select(Program.alphabetical.pluck(:name, :id), @organization.program_ids),
```

Comment:
nice

Reply:
I think I likely got this from your or Tamara's original code ;-)